cmd/operator-sdk/test/local.go

```diff
@@ -117,7 +117,13 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	}
 
 	if tlConfig.upLocal && tlConfig.namespace == "" {
-		return fmt.Errorf("must specify a namespace to run in when -up-local flag is set")
+		// valid case: WATCH_NAMESPACE="" in --up-local
```

Comment:
Could not it work without the namespace flag?

Reply:
yes, it can. But, i perfer the behavior to be explicit. I want "WATCH_NAMESPACE=""` modes to kick in when user is aware of it. 

AFAIK, in already existing, `operator-sdk up local` the behavior of the `--namespace` flag is as floows

`no-flag specified`: sets `WATCH_NAMESPACE` env to namepace from kubecontext (usually defaults to `"default"`)
`--namespace=foo`: sets `WATCH_NAMESPACE` env to `foo`
`--namespace=""`: sets `WATCH_NAMESPACE` env to `""`

my goal is to provide this behavior. 
at present if the `--namespace` flag is absent in `test` command it  sets `WATCH_NAMESPACE` env to namepace from kubecontext https://github.com/operator-framework/operator-sdk/blob/c7f429f05808406068b1a67f49b0de9e6481f545/pkg/test/framework.go#L73